oak_restricted_kernel/src/lib.rs

```diff
@@ -472,9 +473,15 @@ pub fn start_kernel(info: &BootParams) -> ! {
         derived_key,
     );
 
+    // Ensure new process is not dropped. Safety: the ELF file must be valid.
+    let process = Box::leak(Box::new(
+        // Safety: application must be valid.
+        unsafe { Process::from_application(&application).expect("failed to create process") },
+    ));
+
```

Comment:
In the future we'll hold applications in a static map, with PID. For now leaking them will do.